src/coreclr/src/debug/dbgutil/elfreader.cpp

```diff
@@ -172,6 +170,50 @@ ElfReader::PopulateForSymbolLookup(uint64_t baseAddress)
         return false;
     }
 
+    uint64_t minAddr = std::min<uint64_t>(std::min<uint64_t>((uint64_t)m_gnuHashTableAddr,
+                                                             (uint64_t)m_stringTableAddr),
+                                                             (uint64_t)m_symbolTableAddr);
+    uint64_t maxAddr = std::max<uint64_t>(std::max<uint64_t>((uint64_t)m_gnuHashTableAddr + sizeof(GnuHashTable) + sizeof(int32_t),
```

Comment:
Don't you also need to account for the size of the symbol table as well? Also, what's the additional int32 at the end of the GNU hash?

Reply:
I don't have all the size info.  So this is approximate.  It is underestimating the `maxAddr`. The hash is a hash table.  A series of buckets and a series of chains.  The `sizeof(int)` was the absolute minimum number of buckets.  It probably grossly underestimates the `maxAddr`.